charts/redis-ha/README.md

```diff
@@ -0,0 +1,82 @@
+# Redis
+
+[Redis](http://redis.io/) is an advanced key-value cache and store. It is often referred to as a data structure server since keys can contain strings, hashes, lists, sets, sorted sets, bitmaps and hyperloglogs.
+
+## TL;DR;
+
+```bash
+$ helm install stable/redis-ha
+```
+
+By default this chart install one master pod containing redis master container and sentinel container, 2 sentinels and 1 redis slave.
+
+## Introduction
+
+This chart bootstraps a [Redis](https://redis.io) highly available master/slave configuration that is managed by Redis Sentinel. 
+
+## Prerequisites
+
+- Kubernetes 1.5+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+
+## Installing the Chart
+
+To install the chart
+
+```bash
+$ helm install stable/redis-ha
+```
+
+The command deploys Redis on the Kubernetes cluster in the default configuration. By default this chart install one master pod containing redis master container and sentinel container along with 2 redis slave pods each containing their own sentinel sidecars. The [configuration](#configuration) section lists the parameters that can be configured during installation.
```

Comment:
installs 😬

Reply:
Welp looks like @wabernatScality is gonna have his work cut out for him on this one